src/Packs.sol

```diff
@@ -727,6 +738,7 @@ contract Packs is
     /// @dev Only callable by admin role
     /// @dev Emits a MaxPackRewardMultiplierUpdated event
     function setMaxPackRewardMultiplier(uint256 maxPackRewardMultiplier_) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        if (maxPackRewardMultiplier_ == 0) revert InvalidPackRewardMultiplier();
```

Comment:
I think these are lazy dings from the auditor. 

if
` if (maxPackRewardMultiplier_ < minPackRewardMultiplier) revert InvalidPackRewardMultiplier();`

And `minPackRewardMultiplier` is a unsigned int it is impossible for `maxPackRewardMultiplier` to ever be `0`0